backend/app/services/llm/providers/oai.py

```
@@ -28,10 +29,17 @@ def __init__(self, client: OpenAI):
         super().__init__(client)
         self.client = client
 
+    @staticmethod
+    def create_client(credentials: dict[str, Any]) -> Any:
+        if "api_key" not in credentials:
+            return f"OpenAI API Key is not configured"
```

Ruff (0.14.14)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)